File: go/test/exists_test.go
package sdktest

import (
	"testing"

	sdk "github.com/voxgig-sdk/ogli-link-shortener-sdk"
)

func TestExists(t *testing.T) {
	t.Run("test-mode", func(t *testing.T) {
		testsdk := sdk.TestSDK(nil, nil)
		if testsdk == nil {
			t.Fatal("expected non-nil SDK")
		}
	})
}

File: lua/test/exists_test.lua
-- OgliLinkShortener SDK exists test

local sdk = require("ogli-link-shortener_sdk")

describe("OgliLinkShortenerSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
